demo/src/app/app.component.ts

```diff
@@ -191,6 +191,63 @@ import { QueryBuilderClassNames, QueryBuilderConfig } from 'angular2-query-build
 export class AppComponent {
   public queryCtrl: FormControl;
 
+  public getCurrentDatetime():string {
+    let currentDatetime: string;
+    let date: Date = new Date();
+
+    let day: string = ('0'+date.getDate().toString()).slice(-2)
+    let month: string = ('0'+date.getMonth().toString()).slice(-2)
+    let year: string = date.getFullYear().toString();
+    let hours: string = ('0'+(date.getHours()+5).toString()).slice(-2)
+    let minutes: string = ('0'+date.getMinutes().toString()).slice(-2)
+    let seconds: string = ('0'+date.getSeconds().toString()).slice(-2)
+
+    currentDatetime = year +"/"+ month +"/"+ day +" "+ hours +":"+ minutes+":"+seconds;
+    //currentDatetime = date.toISOString();
+    return currentDatetime;
+  }
+
+  // Validates that the input string is a valid date/time formatted as "YYYY/MM/DD HH:MM:SS"
+  public isValidDate(dateString: string)
+  {
+      // First check for the pattern
```

Comment:
This comment should instead mention what the pattern does for those of us who are not great with patterns.